source/hotkeys_.d

```diff
@@ -23,6 +23,8 @@ enum Bindable:char[4]{
 	turnRight="tgrM",
 	cameraZoomIn="imzM",
 	cameraZoomOut="omzM",
+	moveUp="puvM",		//FIXME
+	moveDown="nwdM",	//FIXME
```

Comment:
Should be moved to `SacEngine extensions`.

Reply:
yea, i alrady moved it. it'll be better to re-review after i drop the next one. what about the 4 letters? do i get rid of the FIXME or actually fix them?

Reply:
I think the tags are good, they seem like the canonical thing that Shiny would also have done.

Reply:
done.